src/components/Button.js

```diff
@@ -41,15 +43,13 @@ module.exports = React.createClass({
 		};
 	},
 	render() {
+		var classes = this.props.sheet.classes;
```

Comment:
@kof, Won't pass ESLint validation, since prop `sheet` is not described.

That actually raises the question, why wrapping class into mixin-like wrapper instead of just using styles defined before `var Buttton`?

I know you had this before implemented as mixin, but won't it be better to have styles like:
```js
const styles = require('../styles/button');
const classes = useSheet(styles);

// ...

render() {
  return <div className={classes.container}>
}
```